reference/7.4/Microsoft.PowerShell.Utility/Tee-Object.md

```diff
@@ -97,6 +97,71 @@ drive. A pipeline operator (`|`) sends the list to `Tee-Object`, which appends t
 AllSystemFiles.txt file and passes the list down the pipeline to the `Out-File` cmdlet, which saves
 the list in the `NewSystemFiles.txt file`.
 
+### Example 4: Print output to console and use in the pipeline
+
+This example searches for a metadata key in folder of Markdown documents, prints the current
+setting for the key to the console, then edits the files before printing the updated values.
+
+```powershell
+$consoleDevice = if ($IsWindows) {
+    '\\.\CON'
+} else {
+    '/dev/tty'
+}
+$datePattern = '(ms\.date:\s+)(\d+(\/\d+){2})'
+$newDate = Get-Date -Format 'MM/dd/yyyy'
+
+(Select-String -Path .\About\*.md -Pattern $datePattern) |
+    Tee-Object -FilePath $consoleDevice |
+    ForEach-Object -Process {
+        $filepath    = $_.Path
+        $content = Get-Content -Path $filepath -Raw
+        $content = $content -replace $datePattern, "`${1}$NewDate"
+        Set-Content -Path $filepath -Value $content
+    }
+
+Select-String -Path .\About\*.md -Pattern $datePattern
+```
+
+```Output
+About\about_PSReadLine_Functions.md:6:ms.date: 04/24/2023
+About\about_PSReadLine.md:4:ms.date: 04/24/2023
+
+About\about_PSReadLine_Functions.md:6:ms.date: 04/24/2023
+About\about_PSReadLine.md:4:ms.date: 04/24/2023
```

Comment:
It looks like nothing has changed. The dates are the same.

Reply:
Oops, that's because I ran it twice and didn't notice when I took the output, good catch!